### PR TITLE
Improves jaunt

### PR DESCRIPTION
**Pull request by clusterfack**
-No longer causes 2 deletions!
-You can now see yourself while jaunting! Spoopy 125 alpha, ghost style so you know you are jaunting!
-Fuck you copypasted vampire mist proc!
-Uses standard procs for animations
-Options to enable/disable mist, change animation type

### CODE REVIEW

File: code/modules/spells/targeted/ethereal_jaunt.dm [hunk 1]
```diff
@@ -14,85 +14,66 @@
...
-	animation.icon_state = "liquify"
-	flick("liquify",animation)
+	var/enteranim = "liquify"
+	var/exitanim = "reappear"
+	var/mist = 1
 
-/spell/targeted/ethereal_jaunt/proc/jaunt_reappear(var/atom/movable/overlay/animation, var/mob/living/target)
-	flick("reappear",animation)
+/spell/targeted/ethereal_jaunt/cast(list/targets) //magnets, so mostly hardcoded
+	if(targets.len > 1)
```
Comment (ComicIronic) [inline code/modules/spells/targeted/ethereal_jaunt.dm]: But what's the point, the loop is identical to the single case.

Comment (clusterfack) [inline code/modules/spells/targeted/ethereal_jaunt.dm]: One doesn't need to spawn() to handle it

**Comment by Intigracy** [comment]
phase_shift is the wraith jaunt, right? Because if not, pretty sure that's copypasted code too.

**Comment by clusterfack** [comment]
Fortunately the phase_shift actually inherits the code, instead of copypasting it

**Comment by Arthurdentist** [comment]
Wizards can cast Jaunt even when they are unconscious, though they can't move off the spot they are on. Would be worth looking into that and maybe changing it.

**Comment by clusterfack** [comment]
Should that be changed? We should ask the playerbase

**Comment by Arthurdentist** [comment]
we ought to ask yeah, just something I was thinking about today

**Comment by clusterfack** [comment]
It seemed a majority was in favor of it

**Pull request merged**